### ISSUE TITLE
Columns are truncated in database

### ISSUE THREAD

**Issue by v-tadipatri** [cosminadrianpopescu/vim-sql-workbench#42]
When viewing data in tables, the columns are truncated with "(...)" - according to sql workbench j documentation, there's a "bufferResults=false" option that can be passed on the command line, but if I update the autoload/sw/server.vimto take that as an argument Vim SQL Workbench doesn't work correctly (this is with a Mysql database)- my profiles aren't showing up, and when I try to list the objects, a java stack trace is shown. Is there some way to show the full column values?

**Comment by cosminadrianpopescu** [comment]
There is the option "workbench.console.dataprinter.max.colwidth". By default is set at 150. You can set that to something like 30000. The text in the console mode is always truncated by `sql workbench/j`. So, first solution is to set that property at something really big. But the issue is that if the data is really big, in console is going to look ugly and is going to be hard to follow. I would suggest you to try and export the data in an `ods` format or something similar.

**Comment by v-tadipatri** [comment]
that's exactly what I'm looking for - how is that option set? Do I need to modify the cmd variable in the autoload/sw/server.vim to pass that option? I don't mind the console looking ugly - I'm more concerned about being able to view all the data within the vim sql workbench without exporting it.

**Comment by cosminadrianpopescu** [comment]
You need to set it in the `workbench.settings` file in the profile folder (usually `~/.sqlworkbench`). 
```
workbench.console.dataprinter.max.colwidth = 30000
```

**Comment by cosminadrianpopescu** [comment]
But, please be aware that if you do this, and if the text is too long, the results buffer can become slow and some functions might not work. 

Normally the console is not for viewing long texts. Even in a GUI, you still don't have the full text displayed by default. You still need to do some clicks. It's the same here. Because you can just run your query, then go to run the command `SWSqlExport`. This will export the last executed query. 

So for example, in the buffer, you can execute `select * from table_with_big_data`. Once the result is returned, in the vim command you can do `SWSqlExport`. This will ask you interactively what format you want and where you want to save the resulted file.

**Comment by v-tadipatri** [comment]
that setting worked, but I guess it's whitespace sensitive - I had to remove the spaces before and after the equal sign to get it to work - thank you! I understand that the console is not for viewing long texts, but in those cases where someone has put a giant json string into a field, this setting is incredibly useful.
  I find the "&lt;leader&gt;d" to be really helpful for viewing single records - I wish there was some way to get this command to work in the DBExplorer mode. When there are a lot of columns in the table, it's hard scrolling. Is there some way to get "&lt;leader&gt;d" to work in the DBExplorer?

**Comment by cosminadrianpopescu** [comment]
The same functionality is already available for the db explorer. It's just not mapped. You can select any object on the left panel, click on `D` to view the data. Then Switch to the data panel (in the right), go to a row and call the command `SWDbExplorerToggleFormDisplay`. 

You can map it with something like `nmap <leader>dd :SWDbExplorerToggleFormDisplay<cr>`.
